swim.go

```diff
@@ -165,7 +177,15 @@ func (n *Node) sendUpdates(us []Update) {
 	}
 }
 
-// Updates returns a channel from which Updates can be received.
+// Stop disconnects n from the network and closes its Updates channel.
+func (n *Node) Stop() {
```

Comment:
It seems like this implementation leaks the runReceive goroutine.

Every goroutine needs a plan for how/when it will exit. (In some situations, the plan could be "it never does", but that's prob not a good idea when there's a Stop method on the associated object.)